packages/apps/tools/src/components/Global/RequestKeyField/index.tsx

```diff
@@ -5,7 +5,7 @@ import React, { type FC } from 'react';
 import { FieldError } from 'react-hook-form';
 import * as z from 'zod';
 
-export const RequestLength = { MIN: 43, MAX: 44 };
+export const RequestLength: { MIN: number; MAX: number } = { MIN: 43, MAX: 44 };
```

Comment:
We need to disable this ESLint warning IMO, TS should just infer this 😛

Reply:
Agreed, but I feel like that's a discussion for another time 😅

Reply:
Yeah, approved nonetheless